test/ReporterTest.cpp

```diff
@@ -328,7 +328,7 @@ TEST_F(ReporterTest, generate)
 
     // Other calls
     EXPECT_CALL(m_tree_comm, overhead_send()).WillOnce(Return(678 * 56));
-    EXPECT_CALL(*m_comm, rank()).WillOnce(Return(0));
+    EXPECT_CALL(*m_comm, rank()).WillRepeatedly(Return(0));
```

Comment:
What warranted this change?

Reply:
This is a side effect of the refactor of the Reporter into functions that pass the Comm rather than having a local variable rank that is used repeatedly.